devel/glasskube/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+version             0.16.0
+github.setup        glasskube glasskube $version v
+categories          devel
+maintainers         openmaintainer {esafak @esafak}
+description         Missing Package Manager for Kubernetes
+long_description    Deploy, configure and update Kubernetes packages 20x faster than with Helm
+license             Apache-2
+homepage            https://glasskube.dev/
+
+# Checksums for verification
+checksums           rmd160 dcca396ad2d2b94eae6bad5d851f86271268ff84 \
+                    sha256 c65577b004b0a9d20c27e78c44be415960fe93e34995dba9858db975db83f404 \
+                    size   773135
+
+# Dependencies for build and runtime
+depends_build       port:go port:nodejs22 port:npm10
+
+supported_archs     x86_64 arm64
+
+# Custom build steps
+use_configure       no
+
+build {
+    set ldflags "-s -w -X github.com/glasskube/glasskube/internal/config.Version=${version} \
+                 -X github.com/glasskube/glasskube/internal/config.Date=[clock format [clock seconds] -format %Y-%m-%dT%H:%M:%S%z]"
+
+    # Build the web assets using Node.js
+    system "cd ${worksrcpath} && make web"
+
+    # Build the Go binary
+    system "cd ${worksrcpath} && go build -ldflags \"${ldflags}\" -o ${workpath}/bin/glasskube ./cmd/glasskube"
+}
+
+# Installation phase: install the built binary
+destroot {
+    xinstall -m 755 ${workpath}/bin/glasskube ${destroot}${prefix}/bin/glasskube
+
+    # generate and install the completion scripts
+    file mkdir ${destroot}${prefix}/share/bash-completion/completions
+    file mkdir ${destroot}${prefix}/share/fish/vendor_completions.d
+    file mkdir ${destroot}${prefix}/share/zsh/site-functions
+    system "${destroot}${prefix}/bin/glasskube completion bash > ${destroot}${prefix}/share/bash-completion/completions/glasskube"
+    system "${destroot}${prefix}/bin/glasskube completion fish > ${destroot}${prefix}/share/fish/vendor_completions.d/glasskube"
+    system "${destroot}${prefix}/bin/glasskube completion zsh > ${destroot}${prefix}/share/zsh/site-functions/_glasskube"
```

Comment:
Is there a construct I can use to avoid repeating `${destroot}${prefix}`?

Reply:
Perhaps you could set another variable:

```
destroot {
    ....

    set dest ${destroot}${prefix}

    file mkdir ${dest}/share/....
    system "${dest}/bin/glasskube completion bash > ${dest}/share/....
```